src/test/kotlin/me/spradling/gift/core/tests/unit/api/GetAccountsHandlerTests.kt

```diff
@@ -0,0 +1,68 @@
+package me.spradling.gift.core.tests.unit.api
+
+import io.netty.handler.codec.http.HttpResponseStatus
+import me.spradling.gift.core.api.extensions.unwrap
+import me.spradling.gift.core.api.models.ApiRequest
+import me.spradling.gift.core.api.routes.v1.GetAccountsHandler
+import me.spradling.gift.core.tests.unit.UnitTestBase
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.*
+import org.mockito.Mockito.`when`
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("When I call `handleRequest` on GetAccountsHandler,")
+internal class GetAccountsHandlerTests : UnitTestBase() {
+
+  private val handler = GetAccountsHandler(inMemoryStorageClient)
+
+  @Nested
+  @DisplayName("given valid query parameters,")
+  inner class ValidQueryParameter {
+
+    private val apiAccount = apiConverter.convert(validAccounts["account"]!!)
+
+    @BeforeEach
+    fun setup() {
+//      `when`(mockContext.queryParam("limit")).thenReturn(databaseAccount)
```

Comment:
you could get this to work in the current flow by adding a `list.json` which contains a valid list of Accounts. Ensure they are inserted into the db before the test, and then have a similar API version of  `list.json` and verify that is what you get out.